tests/system/DebugTraceableTraitTest.php

```diff
@@ -28,6 +28,7 @@ final class DebugTraceableTraitTest extends CIUnitTestCase
 {
     public function testFactoryInstanceReturnsWhereItIsRaised(): void
     {
+        /** @phpstan-ignore codeigniter.frameworkExceptionInstance */
```

Comment:
Please upgrade to `coding-standard:v1.8.3`. This can now be written as:
```suggestion
        // @phpstan-ignore codeigniter.frameworkExceptionInstance
```

Reply:
Added to endline